Automattic/jetpack
# WPCOMSH: Removing duplicated functionality to remove masterbar module listing

## On hold

This now may need to wait until the module itself is removed as it seems there may be a priority issue - the module is not disappearing from the modules page using this PR.

Fixes https://github.com/Automattic/vulcan/issues/448

## Proposed changes:

* Once this PR was merged - https://github.com/Automattic/jetpack/pull/38804 - the 'masterbar' module no longer showed up on the Jetpack modules page. As such the function in `wpcomsh/masterbar.php` to do the same is not needed, and is removed in this PR.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/448

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this out, the best option may be to use rsync on a WoA dev site: `jetpack rsync wpcomsh`  - the path being `/srv/htdocs/wp-content/plugins/wpcomsh-dev`
* You may be able to use a WoA test site using the Beta tester plugin and checking out this branch but I was unable to change branches or even change to bleeding edge / RC on `wpcomsh` using the Beta Tester plugin
* Visit the modules page - `wp-admin/admin.php?page=jetpack_modules` - and you should not see any listing for the WordPress.com Toolbar (this is the 'masterbar' module).
